organization_types_categories.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
```

Comment:
i think this is only needed by the OrganizationCreatePage component

Reply:
nope, b-api has to check to make sure purpose / type combination are valid